### PR TITLE
Update data model (#39)

### PR DESCRIPTION
**Pull request by good-move**
Added references between calendars, teams, users, and events

### CODE REVIEW

File: src/lib/model.ts [hunk 1]
```diff
@@ -35,11 +35,13 @@ export interface Organization extends Entity {
   name: string;
   avatar: string;
   description: string;
+  teams: Team[];
 }
 
 export interface Event extends Entity {
   type: string;
   comment: string;
+  createdFor: string;
```
Comment (sergeyt) [inline src/lib/model.ts]: lets rename to calendarId

File: src/lib/model.ts [hunk 2]
```diff
@@ -48,6 +50,7 @@ export interface Calendar extends Entity {
   name: string;
   type: string;
   description: string;
+  teams: Team[];
```
Comment (sergeyt) [inline src/lib/model.ts]: lets asume only one team per calendar for now, also add teamId field, team field should be computed in REST API only

File: src/lib/model.ts (hunk 1 above)
Comment (good-move) [inline src/lib/model.ts]: `createdFor` field was assumed to reference the user, associated with the event. Thus, changes made in the event modal dialog could easily be sent to the server.

**Comment by good-move** [comment]
Changed files according to the review

**Pull request merged**